jupyter-js-widgets/src/utils.js

```diff
@@ -66,14 +66,18 @@ function loadClass(class_name, module_name, registry) {
 
             // If the module is jupyter-js-widgets, we can just self import.
             var modulePromise;
-            if (module_name === 'jupyter-js-widgets') {
-                modulePromise = Promise.resolve(require('../'));
-            } else {
+            if (typeof requirejs !== 'undefined') {
                 modulePromise = new Promise(function(innerResolve, innerReject) {
-                    window.require([module_name], function(module) {
+                    requirejs([module_name], function(module) {
                         innerResolve(module);
                     }, innerReject);
                 });
+            } else if (module_name === 'jupyter-js-widgets') {
+                modulePromise = Promise.resolve(require('../'));
+            } else {
+                // FUTURE: Investigate dynamic loading methods other than require.js.
+                throw new Error(['In order to use third party widgets, you ',
+                    'must have require.js loaded on the page.'].join(''));
```

Comment:
lol, that won't work, you need to check for `module_name === 'jupyter-js-widgets'`first.